config/application.rb

```diff
@@ -29,6 +28,7 @@ class Application < Rails::Application
     config.autoload_paths += Dir[config.root.join('lib', 'middlewares').to_s]
     config.autoload_paths += Dir[config.root.join('lib', 'parsers').to_s]
     config.autoload_paths += Dir[config.root.join('lib', 'renderers').to_s]
+    config.autoload_paths += Dir[config.root.join('lib', 'i18n_scanners').to_s]
```

Comment:
Should we move this line to `config/environment/development.rb` and `config/environment/testing.rb` instead so it only loads in production?